src/index.css

```diff
@@ -1,27 +1,105 @@
 body {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
```

Comment:
- [ ] Kindly ensure you use the recommended font-family type that is Lato or Gill sans.
<img width="762" alt="Screenshot 2022-01-07 at 13 32 18" src="https://user-images.githubusercontent.com/64774936/148544700-583524ba-d375-4815-aabb-fe9d782b90fc.png">

Reply:
Thank you for your feedback, I have taken care of those issues